torch/csrc/autograd/init.cpp

```diff
@@ -574,6 +613,10 @@ static PyMethodDef methods[] = { // NOLINT
   {"get_autocast_cpu_dtype", get_autocast_cpu_dtype, METH_NOARGS, nullptr},
   {"set_autocast_gpu_dtype", set_autocast_gpu_dtype, METH_O, nullptr},
   {"get_autocast_gpu_dtype", get_autocast_gpu_dtype, METH_NOARGS, nullptr},
+  {"set_autocast_xpu_enabled", set_autocast_xpu_enabled, METH_O, nullptr},
+  {"is_autocast_xpu_enabled", is_autocast_xpu_enabled, METH_NOARGS, nullptr},
+  {"set_autocast_xpu_dtype", set_autocast_xpu_dtype, METH_O, nullptr},
+  {"get_autocast_xpu_dtype", get_autocast_xpu_dtype, METH_NOARGS, nullptr},
```

Comment:
Do these public python API's need to be in core? I'm wondering if they should come with the xpu extension, given that the actual XPU autocast kernels live with the extension too.

Reply:
sure, let me remove the public python APIs here.